views/pages/acolhido.ejs

```diff
@@ -49,6 +49,40 @@
             </div>
         </div>
     </div>
-    <form action="<%= acolhido.id %>/prescricao" method="POST">
-        <input class = "botaoPrescricao" type="submit" value="Nova Prescrição">
-    </form>
+
+    <table class="margin-top-5" width="100%">
+        <thead>
+            <tr>
+            <th>ID</th>
+            <th class="text-center">Data</th>
+            <th class="text-center">Validade</th>
+            <th class="text-right">Ação</th>
+            </tr>
+        </thead>
+        <tbody>
+            <%for (prescricao of prescricoes){%>
```

Comment:
No caso do comentário lá de cima, fariamos `for (prescricao of acolhido.prescricaos)`